README.md

```diff
@@ -1,47 +1,74 @@
 # Britecharts React
-Britecharts-react is a React wrapper for the [Britecharts charting library.][britecharts]
+Britecharts-react is a [React][react] wrapper for the [Britecharts][britecharts] charting library.
 
-## Why
-Britecharts is a D3 based, high level charting library that allows users to create stunning data visualizations with little work. [React][react] is one of the most popular User Interface libraries that allows to create apps by composing small components in an intuitive way. Both libraries take control of the DOM, so how can we make them work together in a fluid way?
 
-Britecharts-react proposes a way of integrating Britecharts and React that promotes a *clear separation of concerns* between the libraries and aims for an *easy and effortless maintenance* of the charts.
-
-## Approach
-The React and D3 integration can be done at different levels, leaning more one side or another. We don’t want to get too specific on the details, as for that there is [slides][d3-react] and [a recording][d3-react-video] that explain the different approaches.
+## Usage
+Britecharts-React components are used just like any other stateless React component. You will pass in some props and it would render a chart:
 
-For this project, we have chosen the approach called ‘Mapping Lifecycle methods’ based on [Nicholas Hery's article][integration-article]. You can see some [code example here][integration-article-code]. 
+```
+<TooltipComponent>
+    <StackedAreaComponent
+        data={stackedAreaData.with2Sources()}
+        width={600}
+        height={400}
+    />
+</TooltipComponent>
```

Comment:
We'll have to update this after the render props PR goes in

Reply:
True!